logging/README.md

```diff
@@ -54,9 +54,9 @@ in a directory named `viya4-monitoring-kubernetes-<release_number>`.
 2. From the main page for the repository, click **Code**.
 3. Copy the HTTPS URL for the repository.
 4. From a directory where you want to create the local copy, enter the 
-command `git clone <https_url>`. If you need to use a repeatable process to download a specific release of the repository, include the tag associated with the release in the `git clone` command. The tag for each release is displayed on the Releases page. 
+command `git clone <https_url>`. If you want to download a specific branch, release, or tag, use the `git clone --branch <my_branch> <https_url>` command. Use the `<my_branch>` parameter to specify the branch, release or tag.  If you are developing a repeatable process and need to ensure the same release of the repo is used each time, specify the tag associated with a specific release rather than **stable**. Note that the tag and release names are typically the same, but you should check the Releases page to verify the tag name. 
```

Comment:
Since we tell them to move to the stable branch in Step 1, I think we should update Step 4 to show cloning that particular branch (using the --branch parameter) in the first git clone syntax (in first sentence).

_From a directory where you want to create the local copy, enter the command git clone  --branch stable <https_url>.  You can replace stable with the tag associated with a specific release if you need a version other than the current stable version.  For example, if you are developing a repeatable process and need to ensure the same release of the repo is used every time, specify the tag associated with that specific release rather than stable. Note that the tag and release names are typically the same, but you should check the Releases page to verify the tag name._

Reply:
Revised as noted (both /logging and /monitoring).